larpandoracontent/LArHelpers/LArClusterHelper.cc

```diff
@@ -600,6 +600,48 @@ void LArClusterHelper::GetCoordinateVector(const Cluster *const pCluster, Cartes
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 
+void LArClusterHelper::GetCaloHitList(const Cluster *const pCluster, CaloHitList &caloHitList)
+{
+    for (const OrderedCaloHitList::value_type &layerEntry : pCluster->GetOrderedCaloHitList())
+    {
```

Comment:
I love it.
I'd forgotten about this function; it isn't used anymore but I think it's useful enough generically that it could stay in the helper class.

Done!